tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_Sources.cpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/RadiationTransport/M1Grey/Sources.hpp"
+#include "Evolution/Systems/RadiationTransport/Tags.hpp" // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+
+SPECTRE_TEST_CASE("Unit.RadiationTransport.M1Grey.Sources", "[Unit][M1Grey]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{
+      "Evolution/Systems/RadiationTransport/M1Grey"};
+
+  pypp::check_with_random_values<1>(&RadiationTransport::M1Grey::ComputeSources<
```

Comment:
Are there any analytic solutions this could be tested with? If not that's fine, but it would be good to have some sort of analytic test (even if it's only in a limit) to check that the same typo wasn't made twice (we've had that happen)

Reply:
I should be able to make an arbitrary solution up and write analytically what the source term should be, but i don't know of an easy analytical solution that would test all the terms and is physically motivated.